media/transcoder/api/Transcoder.Samples.Tests/TranscoderFixture.cs

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Google;
+using System;
+using System.Net;
+using System.IO;
+using Xunit;
+using Xunit.Sdk;
+using GoogleCloudSamples;
+using Google.Cloud.Video.Transcoder.V1;
+using System.Collections.Generic;
+
+[CollectionDefinition(nameof(TranscoderFixture))]
+public class TranscoderFixture : IDisposable, ICollectionFixture<TranscoderFixture>
+{
+    public string ProjectId { get; }
+    public string ProjectNumber { get; }
+    public string Location { get; }
+
+    private readonly RandomBucketFixture _bucketFixture;
+
+    public string BucketName { get; }
+    public string TestDataPath => Path.GetFullPath("../../../testdata/");
+    public string TestVideoFileName = "ChromeCast.mp4";
+    public string TestOverlayImageFileName = "overlay.jpg";
+    public string InputUri { get; }
+    public string OverlayImageUri { get; }
+    public Job.Types.ProcessingState JobStateSucceeded = Job.Types.ProcessingState.Succeeded;
+    public List<string> jobIds { get; }
+    public List<string> jobTemplateIds { get; }
+    private readonly DeleteJobSample deleteJobSample;
+    private readonly DeleteJobTemplateSample deleteJobTemplateSample;
```

Comment:
```suggestion
    private readonly DeleteJobSample _deleteJobSample = new DeleteJobSample();
    private readonly DeleteJobTemplateSample _deleteJobTemplateSample = new DeleteJobTemplateSample();
```

Reply:
Done

Reply:
Should I do this in the individual test files as well?